dependencies.yml

```diff
@@ -17,6 +17,9 @@ com.google.guava:
     - com.google.code.findbugs:jsr305
     - com.google.errorprone:error_prone_annotations
 
+com.puppycrawl.tools:
+  checkstyle: { version: !!str 7.7 }
```

Comment:
Like in the spring-boot PR, what do you think about using `'7.7'` as the pattern for strings in this file? I think it's simpler than `!!str`.